tasks/linter.py

```diff
@@ -309,35 +309,49 @@ def list_ssm_parameters(_):
 
 
 @task
-def ssm_parameters(ctx):
+def ssm_parameters(ctx, mode="all", folders=None):
     """
     Lint SSM parameters in the datadog-agent repository.
     """
-    lint_folders = [".circleci", ".github", ".gitlab", "tasks", "test"]
+    modes = ["env", "wrapper", "all"]
+    if mode not in modes:
+        raise Exit(f"Invalid mode: {mode}. Must be one of {modes}")
+    if folders is None:
+        lint_folders = [".circleci", ".github", ".gitlab", "test"]
+    else:
+        lint_folders = folders.split(",")
     repo_files = ctx.run("git ls-files", hide="both")
     error_files = []
     for filename in repo_files.stdout.split("\n"):
         if any(filename.startswith(f) for f in lint_folders):
-            matched = is_get_parameter_call(filename)
-            if matched:
-                error_files.append(matched)
+            calls = list_get_parameter_calls(filename)
+            if calls:
+                error_files.extend(calls)
+    if mode == "env":
+        error_files = [f for f in error_files if not f.with_env_var]
+    elif mode == "wrapper":
+        error_files = [f for f in error_files if not f.with_wrapper]
     if error_files:
-        print("The following files contain unexpected syntax for aws ssm get-parameter:")
+        print(
+            f"[{color_message('ERROR', Color.RED)}] The following files contain unexpected syntax for aws ssm get-parameter:"
+        )
         for filename in error_files:
             print(f"  - {filename}")
         raise Exit(code=1)
+    print(f"[{color_message('OK', Color.GREEN)}] All files are correctly using wrapper for aws ssm parameters.")
 
 
 class SSMParameterCall:
-    def __init__(self, file, line_nb, with_wrapper=False, with_env_var=False):
+    def __init__(self, file, line_nb, with_wrapper=False, with_env_var=False, not_standard=False):
         self.file = file
         self.line_nb = line_nb
         self.with_wrapper = with_wrapper
         self.with_env_var = with_env_var
+        self.not_standard = not_standard
```

Comment:
nit - I'd not used `not` parameters because some people can struggle with the meaning of true/false since it's the opposite

Reply:
Agreed